types/Environment.js

```diff
@@ -31,6 +31,8 @@ declare class $JestEnvironment {
   },
   testFilePath: string,
   moduleMocker: ModuleMocker,
+  setup(): Promise<Global>,
```

Comment:
Can you change the signature to Promise<void>? I don't think we need to return the global here.